tutorials/3d/ragdoll_system.rst

```diff
@@ -0,0 +1,92 @@
+.. _doc_ragdoll_system:
+
+Ragdoll system
+===============
+
+Introduction
+------------
+
+A ragdoll is the use of physics body's simulation to control the bone of a mesh. It is used for death animations in many games and other kind of physics-based animation. Godot has a built-in ragdoll system. We will be using the Platformer3D demo to demonstrate the set up of a ragdoll.
+
+.. note:: You can download the Platformer3D demo on `GitHub <https://github.com/godotengine/godot-demo-projects/tree/master/3d/platformer>` or `the Asset Library <https://godotengine.org/asset-library/asset/125>`_.
+
+Setting up the ragdoll
+--------------------
+
+Create physical bones
+~~~~~~~~~~~~~~~~~~~~~
+
+Similar to many godot's features, there is a node for setting up a ragdoll. You will be using a :ref:`PhysicalBone <class_PhysicalBone>` node. To simplify the set up, you can create the required physical bones with the feature "Create physical skeleton" in the skeleton node.
+
+In the demo, select the skeleton node (in the player scene of Robi the robot). You will notice that a skeleton button appears on the top bar menu:
+
+.. image:: img/ragdoll_menu.png
+
+Click it and select the Create physical skeleton option. The engine will automatically generate physical bones (with their own shape) for each bone of the skeleton and pin joints for connecting the physical bones.
+
+.. image:: img/ragdoll_bones.png
+
+Clean up the skeleton
+~~~~~~~~~~~~~~~~~~~~~
+
+In the picture above, there are unnecessary physical bones like the MASTER bone, your next step is to clean up our skeleton by removing these physical bones.
+
+.. note:: The rule is to remove all physical bones that are too small to make a difference in the simulation and all utility bones (like the MASTER bone in this case).
+
+.. note:: For example, if we take a humanoid skeleton where the hand is comprised of many bones, it's really useless and wasteful to maintain all bones of the hand. One important thing to do in this case is to remove all physical bones and replace them by one or two bones to simulate the hand.
```

Comment:
In general I would recommend to avoid neighboring note blocks. If you have two annotations, you might just use two regular paragraphs as well. Notes means side blocks from the article